app/assets/javascripts/discourse/components/about-page-users.js.es6

```diff
@@ -0,0 +1,45 @@
+import computed from "ember-addons/ember-computed-decorators";
+import { userPath } from "discourse/lib/url";
+import { formatUsername, escapeExpression } from "discourse/lib/utilities";
+import { normalize } from "discourse/components/user-info";
+import { renderAvatar } from "discourse/helpers/user-avatar";
+
+export default Ember.Component.extend({
+  @computed("users")
+  html(users) {
+    let html = "";
+    users.forEach(user => {
+      let name = "";
+      if (user.name && normalize(user.username) !== normalize(user.name)) {
+        name = user.name;
+      }
+      html += `
+        <div data-username="${user.username}" class="user-info small">
+          <div class="user-image">
+            <div class="user-image-inner">
+              <a href="${userPath(user.username)}" data-user-card="${
```

Comment:
would it make any change to store the result of `userPath(user.username)` as you use it two times ?